### DISCUSSION TITLE
When I opened a long image, a scroll bar appeared, but it was not loaded from the top

### DISCUSSION THREAD

**Discussion by zhrrrr** [fancyapps/ui#581] [Q&A] [answered]
When I opened a long image, a scroll bar appeared, but it was not loaded from the top

**Comment by fancyapps** [comment, marked as answer]
Hi, 

Can you, please, show a live page or create a demo using any code playground?

**Reply by zhrrrr** [reply]
Sorry, there are no online cases. I tried to make the following modifications, but after dragging the image, it still returned to its default position

Fancybox.bind("[data-fancybox]", {
            Images: {
                initialSize: 'cover',
                zoom: false,
            },
            on: {
                reveal: (fancybox, slide) => {
                    console.log('fancybox',fancybox)
                    console.log('slide', slide.contentEl.style.filter)
                    slide.contentEl.style.transform = 'matrix(6.1688, 0, 0, 6.1688, 0.008, 3220.09)'
                }
            }
    });

**Reply by fancyapps** [reply]
Please, create a demo using any code playground, there are plenty of them - jsfiddle, codesandbox, codepen, stackblitz, etc, etc. Also, you can easily fork any demo you can find on the docs.
